### PR TITLE
Add reply button to message actions

### PR DESCRIPTION
**Pull request by nishimaki10**
@RocketChat/core

Mention + Quote

ss1

ss2

### CODE REVIEW

**Comment by geekgonecrazy** [comment]
lgtm I actually find my self doing this manually very frequently

File: packages/rocketchat-lib/client/MessageAction.coffee [hunk 1]
```diff
@@ -79,6 +79,33 @@ Meteor.startup ->
...
+		i18nLabel: 'Reply'
+		context: [
+			'message'
+			'message-mobile'
+		]
+		action: (event, instance) ->
+			message = @_arguments[1]
+			input = instance.find('.input-message')
+			url = RocketChat.MessageAction.getPermaLink(message._id)
+			console.log message
```
Comment (rodrigok) [inline packages/rocketchat-lib/client/MessageAction.coffee]: Can you remove this log?

File: packages/rocketchat-lib/client/MessageAction.coffee [hunk 2]
```diff
@@ -79,6 +79,33 @@ Meteor.startup ->
...
+			text = '[ ](' + url + ') @' + message.u.username + ': '
+			if input.value
+				input.value += if input.value.endsWith(' ') then '' else ' '
+			input.value += text
+			input.focus()
+			$(input).keyup()
```
Comment (rodrigok) [inline packages/rocketchat-lib/client/MessageAction.coffee]: Why do we need this keyup?

File: packages/rocketchat-lib/client/MessageAction.coffee [hunk 3]
```diff
@@ -79,6 +79,33 @@ Meteor.startup ->
...
+			RocketChat.MessageAction.hideDropDown()
+		validation: (message) ->
+			room = RocketChat.models.Rooms.findOne({ _id: message.rid })
```
Comment (rodrigok) [inline packages/rocketchat-lib/client/MessageAction.coffee]: The array of usernames will be removed soon (PR #3918)
Can you replace this check by 
```coffeescript
		validation: (message) ->
			if not RocketChat.models.Subscriptions.findOne({ rid: message.rid })?
				return false

			return true
```


Like we are doing here https://github.com/RocketChat/Rocket.Chat/blob/improvements/more-streams/packages/rocketchat-lib/client/MessageAction.coffee#L183-L187

File: packages/rocketchat-lib/client/MessageAction.coffee (hunk 1 above)
Comment (nishimaki10) [inline packages/rocketchat-lib/client/MessageAction.coffee]: Oh. I forgot to remove. thanks.

File: packages/rocketchat-lib/client/MessageAction.coffee (hunk 2 above)
Comment (nishimaki10) [inline packages/rocketchat-lib/client/MessageAction.coffee]: Hmm.. I have copied from the [quote action](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-lib/client/MessageAction.coffee#L215). I will try checking.

Comment (nishimaki10) [inline packages/rocketchat-lib/client/MessageAction.coffee]: This is probably unnecessary. I remove this.

**Comment by rodrigok** [comment]
LGTM

**Pull request merged**